IdentityCore/tests/automation/ui_app_lib/MSIDClearCookiesTestAction.m

```diff
@@ -57,10 +57,14 @@ - (void)performActionWithParameters:(__unused NSDictionary *)parameters
 
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->